shopinvader_api_payment_cart/routers/cart.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 ACSONE SA (https://acsone.eu).
+# @author Stéphane Bidoul <stephane.bidoul@acsone.eu>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from typing import Annotated
+
+from fastapi import Depends
+
+from odoo import api
+
+from odoo.addons.fastapi.dependencies import odoo_env
+from odoo.addons.payment import utils as payment_utils
+from odoo.addons.shopinvader_api_cart.routers import cart_router
+from odoo.addons.shopinvader_api_payment.schemas import PaymentData
+
+
+@cart_router.get("/cart/payable")
```

Comment:
```suggestion
@cart_router.get("/payable")
```

The routes in the cart router don't have the /cart prefix. This is an annoyance that we *may* want to fix but in another PR.